bpdm-gate/src/test/kotlin/org/eclipse/tractusx/bpdm/gate/controller/BusinessPartnerControllerIT.kt

```diff
@@ -51,15 +59,35 @@ import java.time.Instant
 class BusinessPartnerControllerIT @Autowired constructor(
     val testHelpers: DbTestHelpers,
     val gateClient: GateClient,
+    val objectMapper: ObjectMapper
 ) {
+    companion object {
+        const val ORCHESTRATOR_REQUEST_NEW_CLEANING = "/api/golden-record-tasks"
+
+        @JvmField
+        @RegisterExtension
+        val gateWireMockServer: WireMockExtension = WireMockExtension.newInstance()
+            .options(WireMockConfiguration.wireMockConfig().dynamicPort())
+            .build()
+
+        @JvmStatic
+        @DynamicPropertySource
+        fun properties(registry: DynamicPropertyRegistry) {
+            registry.add("bpdm.orchestrator.base-url") { gateWireMockServer.baseUrl() }
+        }
+    }
 
     @BeforeEach
     fun beforeEach() {
         testHelpers.truncateDbTables()
+        gateWireMockServer.resetAll()
+        this.mockOrchestratorApi()
     }
 
     @Test
     fun `insert minimal business partner`() {
+
+
         val upsertRequests = listOf(BusinessPartnerNonVerboseValues.bpInputRequestMinimal)
         val upsertResponses = gateClient.businessParters.upsertBusinessPartnersInput(upsertRequests).body!!
         assertUpsertResponsesMatchRequests(upsertResponses, upsertRequests)
```

Comment:
That's alright for keeping the existing tests running. I would, however, expect tests which check the correct sharing state of the business partners. (Sharing State type is correct and taskIds are as expected)

Either as part of the same tests or other tests just for that check